From 67043f41dd1d375ae04a1f7195b398f5721dfc09 Mon Sep 17 00:00:00 2001
From: Nick Simonov <nicksimonovv@gmail.com>
Date: Tue, 15 May 2018 10:33:52 -0700
Subject: [PATCH] Input: replace hard coded string with __func__ in pr_err()
Git-commit: 67043f41dd1d375ae04a1f7195b398f5721dfc09
References: git-fixes
Patch-mainline: v4.18-rc1

Change hardcoded string "input_set_capability" in pr_err() function call,
replace it with "%s" __func__ instead.

Signed-off-by: Nick Simonov <nicksimonovv@gmail.com>
Signed-off-by: Dmitry Torokhov <dmitry.torokhov@gmail.com>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/input/input.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/input/input.c b/drivers/input/input.c
index 9785546420a7..6365c1958264 100644
--- a/drivers/input/input.c
+++ b/drivers/input/input.c
@@ -1943,8 +1943,7 @@ void input_set_capability(struct input_dev *dev, unsigned int type, unsigned int
 		break;
 
 	default:
-		pr_err("input_set_capability: unknown type %u (code %u)\n",
-		       type, code);
+		pr_err("%s: unknown type %u (code %u)\n", __func__, type, code);
 		dump_stack();
 		return;
 	}
-- 
2.39.0

